src/components/account-header/account-header.yaml

```diff
@@ -142,6 +142,7 @@ examples:
       classes: 'the-header-classes'
   visualRegressionTests:
     backstopScenarioOptions:
+      delay: 2000
```

Comment:
I'm not sure if the delay happens before or after the "beforeTakingScreenshot" interactions - if it's after then we're all good, if it's before then we might need to add a waitFor just after clicking actions are taken in the alternateStates
```
      account-menu-dropdown-open:
        beforeTakingScreenshot:
          - click: ".hmrc-account-menu__link[aria-expanded='false']"
          - waitFor: ".main-nav-is-open"
 ```